### PR TITLE
Add bab screen

### PR DESCRIPTION
**Pull request by Stephanie-Delicia**

### CODE REVIEW

**Comment by tomgrbz** [comment]
Looks great!

**Comment by Stephanie-Delicia** [comment]
> Looks great!

Sorry to bother you again, but I have a final request. I tried pulling from the new add bab api branch and attempted to implement the add bab functionality here, but I get a specific response error which I included in the README.

**Review by allychoung — COMMENTED** [review]
Can we add a Toast popup reporting on the status of the new bab? Something like "Bab created"

**Comment by Stephanie-Delicia** [comment]
> Can we add a Toast popup reporting on the status of the new bab? Something like "Bab created"

That would be nice to have. I can look into it tonight!

**Comment by Stephanie-Delicia** [comment]
About that conflicting file: I believe the BabApi on main needs to overwritten by the BabApi on this branch

edit: you also need to log in because this screen needs the logged in user information

File: app/src/main/java/com/example/cs4520_twitter/data_layer/api/models/AddBabRequest.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package com.example.cs4520_twitter.data_layer.api.models
+
+data class AddBabRequest(val content: String)
```
Comment (allychoung) [inline app/src/main/java/com/example/cs4520_twitter/data_layer/api/models/AddBabRequest.kt:3]: This is just a question. Does the api handle the date and default like count or would we have to add this in here?

**Review by allychoung — APPROVED** [review]

**Pull request merged**